x-pack/plugins/ml/public/application/explorer/anomaly_timeline.tsx

```diff
@@ -211,6 +222,7 @@ export const AnomalyTimeline: FC<AnomalyTimelineProps> = React.memo(
           <EuiSpacer size="m" />
 
           <SwimlaneContainer
+            id="overall"
```

Comment:
When there are no scores > 0 in the Overall swimlane for the selected time range, it fails to render with the following error shown in the console:

![image](https://user-images.githubusercontent.com/7405507/95216956-947e9f00-07ea-11eb-8699-8acc5c370d29.png)

Reply:
Fixed in https://github.com/elastic/kibana/pull/79315/commits/6e6321582c7ce3c830749aa66a81e1dc09a24355

Reply:
Confirmed fixed by https://github.com/elastic/kibana/pull/79315/commits/6e6321582c7ce3c830749aa66a81e1dc09a24355. I now see:

![image](https://user-images.githubusercontent.com/7405507/95230779-35c12180-07fa-11eb-9d7d-26e8bcbbe5b8.png)